lib/dfe/analytics/entity_table_check_job.rb

```diff
@@ -12,15 +12,15 @@ def perform
         return unless DfE::Analytics.entity_table_checks_enabled?
 
         DfE::Analytics.entities_for_analytics.each do |entity_name|
-          DfE::Analytics.models_for_entity(entity_name).each do |model|
-            entity_table_check_event = build_event_for(model)
-            DfE::Analytics::SendEvents.perform_later([entity_table_check_event])
-            Rails.logger.info("Processing data for #{model.table_name} with row count #{model.count}")
-          end
+          entity_table_check_event = build_event_for(entity_name)
```

Comment:
We can add the processing message back here. It was useful for debugging I found. Suggestion:

```suggestion
          entity_table_check_event = build_event_for(entity_name)
          Rails.logger.info("DfE::Analytics Processing entity: #{entity_name}: Row count #{entity_table_check_event[:row_count]}") if entity_table_check_event[:row_count].present?
          
```

Reply:
I had to go for `row_count = entity_table_check_event['data'].find { |item| item['key'] == 'row_count' }['value'].first` 😢